src/Servers/Connections.Abstractions/ref/Microsoft.AspNetCore.Connections.Abstractions.netcoreapp.cs

```diff
@@ -123,8 +123,58 @@ public partial interface IConnectionListenerFactory
     {
         System.Threading.Tasks.ValueTask<Microsoft.AspNetCore.Connections.IConnectionListener> BindAsync(System.Net.EndPoint endpoint, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken));
     }
-    public partial interface IMultiplexedConnectionListenerFactory : Microsoft.AspNetCore.Connections.IConnectionListenerFactory
+    public partial interface IMultiplexedConnectionBuilder
     {
+        System.IServiceProvider ApplicationServices { get; }
+        Microsoft.AspNetCore.Connections.MultiplexedConnectionDelegate BuildMultiplexed();
+        Microsoft.AspNetCore.Connections.IMultiplexedConnectionBuilder UseMultiplexed(System.Func<Microsoft.AspNetCore.Connections.MultiplexedConnectionDelegate, Microsoft.AspNetCore.Connections.MultiplexedConnectionDelegate> middleware);
```

Comment:
Same feedback here

Reply:
Also, ListenOptions.Use most of the time will be ambiguous:
``` 
The call is ambiguous between the following methods or properties: 'ConnectionBuilderExtensions.Use(IConnectionBuilder, Func<ConnectionContext, Func<Task>, Task>)' and 'MultiplexedConnectionBuilderExtensions.Use(IMultiplexedConnectionBuilder, Func<MultiplexedConnectionContext, Func<Task>, Task>)'
```